data/mods/Xedra_Evolved/items/guns.json

```diff
@@ -86,11 +86,11 @@
     "dispersion": 358,
     "durability": 8,
     "modes": [ [ "DEFAULT", "burst", 10 ] ],
-    "ups_charges": 1,
+    "energy_drain": "1 kJ",
     "reload": 200,
     "default_mods": [ "bars" ],
     "valid_mod_locations": [ [ "accessories", 4 ], [ "sights", 1 ], [ "rail mount", 1 ] ],
-    "flags": [ "MOUNTED_GUN" ],
+    "flags": [ "MOUNTED_GUN", "USE_UPS" ],
```

Comment:
I have tried to follow along but I want to ask to make sure I understand.   "USE_UPS" is what allows the turret to draw energy from the vehicle battery when it fires?   This is a turret only gun so I want to make sure all it needs is to be on a vehicle turret mount, with a battery with sufficient energy and not a UPS.